drivers/watchdog/Kconfig

```diff
@@ -14,6 +14,12 @@ menuconfig WATCHDOG
 
 if WATCHDOG
 
+config WDT_0_NAME
```

Comment:
Some drivers get this from dts, so you need to condition the config on !HAS_DTS_WDT

Reply:
Done.